ci/snapshot/cloudfront.yaml

```
@@ -0,0 +1,188 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters:
+
+  S3BucketProofs:
+    Type: String
+    Description: "S3 bucket will be AccountId-Region-S3BucketSuffix"
+
+Resources:
+
+  S3AccessIdentity:
+    Type: AWS::CloudFront::CloudFrontOriginAccessIdentity
+    Properties:
+      CloudFrontOriginAccessIdentityConfig:
+        Comment: "S3 Bucket access identity"
+
+  TheBucketPolicy2:
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      Bucket: !Ref S3BucketProofs
+      PolicyDocument:
+        Statement:
+          - Action:
+              - s3:GetObject
+            Effect: Allow
+            Resource: !Join [ "", [ "arn:aws:s3:::", !Ref S3BucketProofs, "/*/out/*" ] ]
+            Principal:
+              CanonicalUser: !GetAtt S3AccessIdentity.S3CanonicalUserId
+
+
+  LambdaEdgeFunctionSample:
+      Type: AWS::Lambda::Function
+      Properties:
+        Role: !GetAtt LambdaEdgeFunctionRole.Arn
+        Runtime: python3.7
+        Handler: index.lambda_handler
+        Timeout: 5
+#        AutoPublishAlias: live
+        Code:
+          ZipFile: !Sub |
+            import boto3
+            import logging
+            logger = logging.getLogger('CloudFrontLambdaEdge')
+            logger.setLevel(logging.INFO)
+            PROOFS_BKT = "${S3BucketProofs}"
+            def lambda_handler(event, context):
+              request = event['Records'][0]['cf']['request']
```

This code doesn't seem to do much error handling. Is that typical? We just let it crash and look in the logs?

---

OK